File: Solutions/26_Remove_Duplicates_from_Sorted_Array/Python3/Solution01.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if(len(nums) == 0): return 0

        result = 0
        j = 0

        for i in range(1, len(nums)):
            if(nums[j] != nums[i]):
                nums[j+1] = nums[i]
                j += 1
                result += 1
        
        return result + 1

File: Solutions/0026_Remove_Duplicates_from_Sorted_Array/C/Solution01.c
int removeDuplicates(int* nums, int numsSize) {
    if(numsSize == 0) return 0;

    int result = 0, j = 0;
    for(int i = 1; i < numsSize; ++i){
        if(nums[i] != nums[j]){
            nums[++j] = nums[i];
            ++result;
        }
    }
    return result + 1;
}